resources/web/mobileAppStudy/panel/studySetup.js

```diff
@@ -1,290 +0,0 @@
-/*
```

Comment:
Not too familiar with ext architecture—is there more I ought to delete?

Reply:
Would be worthwhile to get someone more familiar with JavaScript / Ext4 to do a quick review of the client code. Nick, Alan, Cory, etc.

Reply:
Checked in with Nick and we're looking good 👍